include/af/statistics.h

```diff
@@ -43,6 +43,8 @@ AFAPI array mean(const array& in, const dim_t dim=-1);
 AFAPI array mean(const array& in, const array& weights, const dim_t dim=-1);
 
 /**
+   \deprecated Use \ref af::varV2 instead
```

Comment:
This is supposed to be at the bottom of the comment. It is like that in all other places in the headers.

Reply:
I understand consistency concern but ideally I would expect to see if a function is deprecated right at the front of it's documentation rather than towards end after spending time reading it.